### PR TITLE
Hugo/editor: Redesign Info Blocks

### PR DESCRIPTION
**Pull request by anne-usmedia**
- New info blocks styling
- Added exclamation mark svg
- Adjusted info blocks example page
- Added new color red to hugo and the editor
- Removed normal font weight from h4-h6

For https://linear.app/usmedia/issue/CUE-121

### CODE REVIEW

**Review by Lyoness — APPROVED** [review]
Looks great.  Importing into Gerrit.

**Comment by Lyoness** [comment]
https://review.gerrithub.io/c/cue-lang/cuelang.org/+/550731

File: hugo/assets/scss/components/note.scss [hunk 1]
```diff
@@ -1,31 +1,62 @@
 @import '../config/colors';
 @import '../mixins/marginless-child';
+@import '../mixins/screen';
+@import '../mixins/svg';
 
 .note {
     $self: &;
 
-    --note-bg-color: #{ $c-blue--lighter };
```
Comment (JorindeUsMedia) [inline hugo/assets/scss/components/note.scss:7]: for themed components it's best to set the variables at the top for the default colors so it's clear what variables are themed. So these should not be removed here. For themed components we set the color variables at the top.

File: hugo/assets/scss/components/note.scss [hunk 2]
```diff
@@ -1,31 +1,62 @@
...
         --note-bg-color: #{ $c-red--light };
-        --note-border-color: #{ $c-red };
+    }
+
+    @include screen($screen-simple) {
+        &__content {
+            padding-left: 3rem;
+
+            &::before {
+                @include svg(exclamation, $c-white);
```
Comment (JorindeUsMedia) [inline hugo/assets/scss/components/note.scss:43]: Is it possible to render the icons inside the html? Or can it be only done via css?

Comment (JorindeUsMedia) [inline hugo/assets/scss/components/note.scss:43]: best to force it to be a string 'exclamation'

**Review by JorindeUsMedia — CHANGES_REQUESTED** [review]

**Comment by Lyoness** [comment]
@anne-usmedia please rebase this PR to resolve merge conflicts

**Comment by anne-usmedia** [comment]
@Lyoness done

**Comment by myitcv** [comment]
Thanks @anne-usmedia - reimporting now.

**Comment by Lyoness** [comment]
Closes [846eb8b](https://github.com/cue-lang/cuelang.org/commit/846eb8b7dca132cd6e1f80ffdef6c44a3a44eeb4) as merged.

**Pull request closed without merging**